src/category_theory/monoidal/functor.lean

```diff
@@ -190,6 +192,8 @@ def comp : monoidal_functor.{v₁ v₃} C E :=
   μ_is_iso := by { dsimp, apply_instance },
   .. (F.to_lax_monoidal_functor).comp (G.to_lax_monoidal_functor) }.
 
+infixr ` ⊗⋙ `:80 := comp -- We overload notation; potentially dangerous, but it seems to work.
```

Comment:
I feel like this could get annoying to debug if something isn't working...

Reply:
My preference would be to run with it, but I'm happy to be overruled. (We do currently have `\otimes` overloaded for objects, morphisms, and isomorphisms, and so far that seems pretty sane.) If anyone says: "but we have a general rule not to overload notation in mathlib, as a general principle" then I'll believe them. :-)